### PR TITLE
Add feature test for new album

### PR DESCRIPTION
**Pull request by sawleha16**

### CODE REVIEW

File: spec/features/new_album_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  context 'Create new album' do
+    before do
+      create(:user, email: 'abc@gmail.com')
+    end
+
+    specify  do
+      visit root_path
+      expect(page).to have_current_path('/users/sign_in')
+        user_login_form('abc@gmail.com', '1234test')
```
Comment (ahmad19) [inline spec/features/new_album_spec.rb]: should be below expect

File: spec/features/new_album_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      visit "/albums/new"
+      expect do
+        album_form('party','xyz xyz')
+      end.to change(Album, :count).by(1)
+    end
+
+    specify do
+      visit root_path
+      expect(page).to have_current_path('/users/sign_in')
+        user_login_form('abc@gmail.com', '1234test')
```
Comment (ahmad19) [inline spec/features/new_album_spec.rb:38]: should be below expect

**Review by ahmad19 — APPROVED** [review]

File: spec/features/new_album_spec.rb (hunk 2 above)
Comment (sawleha16) [reply spec/features/new_album_spec.rb:38]: Done

File: spec/features/new_album_spec.rb (hunk 1 above)
Comment (sawleha16) [reply spec/features/new_album_spec.rb]: Done

**Pull request merged**